### PR TITLE
Chore: Remove unused variables and cleanup

### PR DESCRIPTION
**Pull request by BigRoy**
## Changelog Description

Removing some unused variables. In some cases the unused variables _seemed like they should've been used - maybe?_ so please **double check the code whether it doesn't hint to an already existing bug**.

Also tweaked some other small bugs in code + tweaked logging levels.

## Additional info

Some general cleanup for fun.
Commit messages also should explain some of the changes made.

This must make for a fun review because Github itself doesn't highlight at all whether the variable is actually unused or not.

## Testing notes:

1. Read the code.
2. Potentially check the areas it touches?

### CODE REVIEW

**Review by iLLiCiTiT — APPROVED** [review]
All changes looks good to me. @antirotor could you double check it please?

File: openpype/hosts/flame/plugins/load/load_clip_batch.py [hunk 1]
```diff
@@ -45,7 +45,6 @@ def load(self, context, name, namespace, options):
         self.batch = options.get("batch") or flame.batch
 
         # load clip to timeline and get main variables
-        namespace = namespace
```
Comment (jakubjezek001) [inline openpype/hosts/flame/plugins/load/load_clip_batch.py:48]: haha! nice 👍

**Review by jakubjezek001 — APPROVED** [review]
LGTM!

**Review by moonyuet — APPROVED** [review]
Looks good for me in Max and Houdini

**Review by antirotor — APPROVED** [review]
I've checked and tested as I could and it looks great!

**Pull request merged**